### PR TITLE
Update SQLiteStudio

### PR DESCRIPTION
**Pull request by JanWerder**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/148858)

### CODE REVIEW

**Comment by JanWerder** [comment]
@microsoft-github-policy-service agree

**Comment by JanWerder** [comment]
> @JanWerder please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

File: manifests/s/sqlitestudio/pl/SQLiteStudio/3.4.4.0/sqlitestudio.pl.SQLiteStudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  InstallerSwitches:
+    Silent: --mode unattended --install_for all
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://github.com/pawelsalawa/sqlitestudio/releases/download/3.4.4/SQLiteStudio-3.4.4-windows-installer.exe
+  InstallerSha256: 39aa488e76fc030a0e34fe797ff7e4ce856a1be31d8881d61dae72b6f3e36da0
+  InstallerSwitches:
+    Silent: --mode unattended --install_for all
+ManifestType: installer
+ManifestVersion: 1.2.0
```
Comment (stephengillie) [inline manifests/s/sqlitestudio/pl/SQLiteStudio/3.4.4.0/sqlitestudio.pl.SQLiteStudio.installer.yaml:20]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/s/sqlitestudio/pl/SQLiteStudio/3.4.4.0/sqlitestudio.pl.SQLiteStudio.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+PublisherUrl: https://www.sqlitestudio.pl/
+PublisherSupportUrl: https://github.com/pawelsalawa/sqlitestudio/issues
+PackageName: SQLiteStudio
+PackageUrl: https://github.com/pawelsalawa/sqlitestudio/releases
+License: GPL-3.0
+LicenseUrl: https://github.com/pawelsalawa/sqlitestudio/blob/master/LICENSE
+Copyright: Copyright sqlitestudio.pl
+ShortDescription: SQLiteStudio is desktop application for browsing and editing SQLite database files. It is aimed for people, who know what SQLite is, or what relational databases are in general.
+ManifestType: defaultLocale
+ManifestVersion: 1.2.0
```
Comment (stephengillie) [inline manifests/s/sqlitestudio/pl/SQLiteStudio/3.4.4.0/sqlitestudio.pl.SQLiteStudio.locale.en-US.yaml:17]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/s/sqlitestudio/pl/SQLiteStudio/3.4.4.0/sqlitestudio.pl.SQLiteStudio.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: sqlitestudio.pl.SQLiteStudio
+PackageVersion: 3.4.4.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.2.0
```
Comment (stephengillie) [inline manifests/s/sqlitestudio/pl/SQLiteStudio/3.4.4.0/sqlitestudio.pl.SQLiteStudio.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @JanWerder,

We've depreciated this manifest version, but most fields are identical to the newest manifest version. So usually, just increasing the number is enough.

**Comment by JanWerder** [comment]
> Hi @JanWerder,
> 
> We've depreciated this manifest version, but most fields are identical to the newest manifest version. So usually, just increasing the number is enough.

Thanks! I'll try that

**Review by stephengillie — APPROVED** [review]

**Pull request merged**